docs/source/developer/contributing.rst

```diff
@@ -92,8 +92,9 @@ When working with a source checkout of Numba you will also need a development
 build of llvmlite. These are available from the ``numba/label/dev`` channel on
 `anaconda.org <https://anaconda.org/numba/llvmlite>`_.
 
-
-Then, to create an environment with a few of the most common dependencies::
+Then, to create an environment with a few of the most common dependencies. Note
+the use of the double-colon syntax (``numba/label/dev::llvmlite``) to install
+the latest development version of the llvmlite library.
```

Comment:
I think "Then" doesn't apply because nothing was done prior. The text also needs a `::` to render the shell command as code:

```suggestion
To create an environment with the required dependencies, noting the use of the 
double-colon syntax (``numba/label/dev::llvmlite``) to install the latest
development version of the llvmlite library::
```

Reply:
<img width="1124" alt="Screen Shot 2023-05-04 at 10 23 26" src="https://user-images.githubusercontent.com/41563/236149731-c2d78d43-afcc-4992-8e47-45d7a6f7fb1b.png">

Looks good.